docs/faq.rst

```diff
@@ -75,3 +75,25 @@ If it crashes with a UnicodeError, what can I do?
 
         export LANG=en_US.UTF-8  # or similar, important is correct charset
 
+Why was borg forked from Attic?
+    borg was created in may 2015 in response to the difficulty of
+    getting new merge requests and larger changes incorporated into
+    attic. more details can be found in the `fairly long discussion
+    <https://github.com/jborg/attic/issues/217>`_ in 
+    attic issue queue that led to the fork.
```

Comment:
I had to read the sentence twice due to your reference to "in attic issue queue". it would be simpler like seen above.